release.sh

```diff
@@ -45,7 +45,7 @@ echo "Go to folder where site was build and sources are already at required tag"
 cd target/checkout
 
 echo "Generating web site"
-mvn -e --no-transfer-progress site
+mvn -e --no-transfer-progress site -Pno-validations -Dmaven.javadoc.skip=false
```

Comment:
Javadoc is a part of the site: https://checkstyle.org/apidocs/index.html
The option `maven.javadoc.skip` will disable generation of API reference.

Reply:
It is skip as FALSE, please look at diff of folders, I prove that API is generated.
Content is same, just timestamp are different